module/vision/BallDetector/data/config/BallDetector.yaml

```diff
@@ -12,7 +12,7 @@ green_ratio_threshold: 0.1
 green_radial_samples: 12
```

Comment:
probably want to change the config paramters to be lower: 1 model, 30 iterations, etc.